src/test/kotlin/nl/info/zac/app/zaak/ZaakKoppelenRestServiceTest.kt

```diff
@@ -160,6 +160,200 @@ class ZaakKoppelenRestServiceTest : BehaviorSpec({
         }
     }
 
+    Given("A source no-link zaak and target hoofdzaak") {
+        val sourceZaak = createZaak(
+            identificatie = "ZAAK-2000-00001",
+            archiefnominatie = Archiefnominatie.BLIJVEND_BEWAREN,
+            zaakTypeURI = zaakTypeURI,
+        )
+
+        val zaakZoekObject = createZaakZoekObject(
+            type = ZoekObjectType.ZAAK,
+            zaaktypeOmschrijving = ZAAK_TYPE_OMSCHRIJVING,
+            identificatie = zoekZaakIdentifier,
+            omschrijving = OMSCHRIJVING,
+            statustypeOmschrijving = STATUS_TYPE_OMSCHRIJVING,
+            zaaktypeUuid = zaakZoekObjectTypeUuid,
+            archiefNominatie = Archiefnominatie.BLIJVEND_BEWAREN.toString()
+        ).apply {
```

Comment:
If you need an apply here why not just extend the `createZaakZoekObject` function with this argument and do the apply in there? We do this for all fixture functions.